monitor/tasks.go

```diff
@@ -50,28 +50,34 @@ func (tm *TaskMonitor) CleanupTasks(ctx context.Context, projects map[string]mod
 		}
 	}
 
-	grip.Info("Done cleaning up tasks")
-
 	return errs
 }
 
 // clean up the passed-in slice of tasks
 func cleanUpTasks(taskWrappers []doomedTaskWrapper, projects map[string]model.Project) []error {
-	grip.Infof("Cleaning up %d tasks...", len(taskWrappers))
+	grip.Info(message.Fields{
+		"runner":  RunnerName,
+		"message": "Cleaning up tasks",
+		"count":   len(taskWrappers),
+		"tasks":   taskWrappers,
```

Comment:
Could this be potentially very large? Should we avoid logging it?

Reply:
oops, thought the struct was just 2 strings